src/TesApi.Web/BatchScheduler.cs

```diff
@@ -965,19 +955,11 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(string taskId, TesT
             //    }
             //}
 
-            var batchExecutionDirectoryPath = GetBatchExecutionDirectoryPath(task);
-            var metricsPath = $"/{batchExecutionDirectoryPath}/metrics.txt";
+            var storageUploadPath = GetStorageUploadPath(task, defaultStorageAccountName);
+            var metricsPath = $"/{storageUploadPath}/metrics.txt";
             var metricsUrl = new Uri(await storageAccessProvider.MapLocalPathToSasUrlAsync(metricsPath, getContainerSas: true));
 
             var additionalInputFiles = new List<TesInput>();
-            // TODO: Cromwell bug: Cromwell command write_tsv() generates a file in the execution directory, for example execution/write_tsv_3922310b441805fc43d52f293623efbc.tmp. These are not passed on to TES inputs.
```

Comment:
I don't think we've verified that this is no longer an issue. I think we need to keep this code in place.